packages/ui/src/ui/containers/Host/Host.scss

```diff
@@ -3,6 +3,18 @@
     overflow: hidden;
     align-items: center;
 
+    &__link {
+        display: inline-block;
+        overflow: hidden;
+        text-overflow: ellipsis;
+        white-space: nowrap;
+        width: 100%;
+    }
+
+    &__tooltip {
+        width: 100%;
```

Comment:
Shouldn't we set all required attributes with `text-overflow: ellipsis` here?
Do we really need width: 100%? Will it work correctly without it?

Reply:
You are right, we don't need to use styles on the link tag.